unicodetools/src/main/java/org/unicode/text/UCA/FractionalUCA.java

```diff
@@ -975,12 +995,12 @@ static void writeFractionalUCA(String filename) throws IOException {
             highByteToScripts.addScriptsIn(np, sample);
             fractionalStatistics.printAndRecord(true, fakeString.next(), np, ns, nt, null);
 
-            longLog.print(
+            output.frac.print(
```

Comment:
ditto

Reply:
same thing, writes a comment which goes to output.frac (former longLog) but not to output.fracShort